src/sass/_variables.scss

```diff
@@ -9,3 +9,10 @@ $primary: #d58e32;
 $text-color: #2a2a2a;
 
 $form-border-color: #292929;
+
+$animation-time-bg: 0.25s;
```

Comment:
Inconsistent variable naming: once you have `animation-time` as a prefix, another time there is as a suffix. I suggest using prefixes - it's easier to keep them together.

Reply:
Done, I've changed naming of the variables. I know that this task isn't quite over 'cause of other tasks being not merged yet. Once they will be merged I will add animation to other hover elements that aren't done so far.